terrariumNotification.py

```diff
@@ -356,6 +356,14 @@ def message(self, message_type, data=None, files=[]):
                 try:
                     # Legacy text formatting using '$' sign
                     text = message.message.replace("${", "{").format(**data)
+                    codelist = re.findall("{.*}",text)
+
+                    logger.info(f"codelist {codelist}")
+
+                    for code in codelist:
+                        resutl = eval(code[1:-1])
```

Comment:
Hm, eval code is pretty bad. So I think it will be saver with `eval(code[1:-1],{"__builtins__": {}}, {})` this should at least not allow importing extra modules: https://realpython.com/python-eval-function/#minimizing-the-security-issues-of-eval

Reply:
also a typo in `result` ;)